packages/cli/src/databases/migrations/common/1740445074052-UpdateParentFolderIdColumn.ts

```diff
@@ -0,0 +1,97 @@
+import type { BaseMigration, MigrationContext } from '@/databases/types';
+
+export class UpdateParentFolderIdColumn1740445074052 implements BaseMigration {
+	async up({
+		escape,
+		queryRunner,
+		copyTable,
+		schemaBuilder: { dropForeignKey, createTable, column },
+		isPostgres,
+		isMysql,
+		tablePrefix,
+	}: MigrationContext) {
+		const workflowTableName = escape.tableName('workflow_entity');
+		const folderTableName = escape.tableName('folder');
+		const parentFolderIdColumn = escape.columnName('parentFolderId');
+		const idColumn = escape.columnName('id');
+
+		if (isMysql) {
+			// Initial migration did not add the foreign key constraint so no need for the delete operation
+			await queryRunner.query(
+				`ALTER TABLE ${workflowTableName} ADD CONSTRAINT fk_workflow_parent_folder FOREIGN KEY (${parentFolderIdColumn}) REFERENCES ${folderTableName}(${idColumn}) ON DELETE CASCADE`,
+			);
+		} else if (isPostgres) {
+			const workflowTable = await queryRunner.getTable(`${tablePrefix}workflow_entity`);
+
+			if (workflowTable) {
+				// Find the foreign key
+				const foreignKey = workflowTable.foreignKeys.find(
+					(fk) =>
+						fk.columnNames.includes('parentFolderId') &&
+						fk.referencedTableName === `${tablePrefix}folder`,
+				);
+
+				if (foreignKey) {
+					await dropForeignKey(
+						'workflow_entity',
+						'parentFolderId',
+						['folder', 'id'],
+						foreignKey.name,
+					);
+
+					await queryRunner.query(
+						`ALTER TABLE ${workflowTableName} ADD CONSTRAINT fk_workflow_parent_folder FOREIGN KEY (${parentFolderIdColumn}) REFERENCES ${folderTableName}(${idColumn}) ON DELETE CASCADE`,
```

Comment:
`workflowTableName` and `folderTableName` missing the tablePrefix here, I suppose have to have that present everywhere in these migrations?

Reply:
escape.tableName returns the name with the prefix.

Reply:
Right, then you can disregard these comments